debian/rules

```diff
@@ -19,6 +19,7 @@ endif
 
 extraopts += -DWITH_OCF=ON -DWITH_LTTNG=ON
 extraopts += -DWITH_PYTHON3=ON -DWITH_MGR_DASHBOARD_FRONTEND=OFF
+extraopts += -DWITH_PYTHON2=OFF -DWITH_MGR_PYTHON3=ON -DMGR_PYTHON_VERSION=3
```

Comment:
i'd recommend:
```sh
extraopts += -DWITH_MGR_DASHBOARD_FRONTEND=OFF
extraopts += -DWITH_PYTHON3=3 -DWITH_PYTHON2=OFF -DMGR_PYTHON_VERSION=3
```

three reasons: 

- group python related options in the same line
- be consistent with specifying python3 versions.
- `WITH_MGR_PYTHON3` is not a valid option at this moment.